sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -12838,16 +12093,16 @@
                                 "value": ""
                               }
                             ],
-                            "content": "CSGD\n <span\n class=\"formio-custom-tooltip\">\n   <span class=\"tooltip-text\">\n    Canada Student Grant for Students with Dependents\n    </span>\n</span>",
+                            "content": "Canada Student Grant for Student with Disabilities (CSG-PD)\n <span\n class=\"formio-custom-tooltip\">\n   <span class=\"tooltip-text\">\n    Please click <a\n            href=\"https://studentaidbc.ca/explore/grants-scholarships/canada-student-grant-students-disabilities\"\n            rel=\"noopener\"\n            target=\"_blank\"\n            >here</a> for more information about this grant.\n    </span>\n</span>",
```

Comment:
Was there an extra effort to try to accommodate the link inside the tooltip?
For form.io component, we can give a small effort but we should try to use the components as much as possible out-of-box.
While hovering over the table the tooltip sometimes is displayed from a different place from where the mouse is.
![image](https://github.com/user-attachments/assets/3dba9684-c458-4787-9a7c-0e5a6e94af7b)

If the solution is accepted by the business it should be ok. For similar situations we can propose an alternative approach, for instance, creating the link in the award acronym or adding a link of type "Click to read more", etc.

Reply:
The effort was just to make the tooltip visible while the user moves the cursor to click on the link.

I was checking the previous code and it seems that it had this behavior before in dev:
![image](https://github.com/user-attachments/assets/a0b498ea-a2df-4f57-9fe0-6458ab413fb7)

Reply:
My point was, did the hyperlink make the goal harder to achieve and demand extra effort?
If yes, as a general recommendation, we do not need to put extra effort into supporting something that is not out-of-box in form.io.
If it was a minor effort it should be ok. I am just making it clear for future developments.
If the current approach is working we do not need to undo it.